core/src/test/groovy/pl/allegro/tech/embeddedelasticsearch/SynchronicitySpec.groovy

```diff
@@ -0,0 +1,38 @@
+package pl.allegro.tech.embeddedelasticsearch
+
+import spock.lang.Specification
+
+import static java.util.concurrent.TimeUnit.MINUTES
+
+class SynchronicitySpec extends Specification {
```

Comment:
Well, this spec proofs nothing :/ You can make spec that will invoke start method second time with assertion noExceptionThrown()

Reply:
the test I've added failed before I wrote the implementation. A test that starts the server a second time from the same thread did not fail...

Reply:
It will fail if you specify same http/transport port :)

Reply:
maybe, but the test as I wrote it is closer to the scenario i'm facing (explained in #64)